src/entity.ts

```diff
@@ -991,16 +991,6 @@ export namespace entity {
     const MAX_DATASTORE_VALUE_LENGTH = 1500;
     if (Array.isArray(entities)) {
       for (const entry of entities) {
-        if (entry && entry.name && entry.value) {
-          if (
-            is.string(entry.value) &&
-            Buffer.from(entry.value).length > MAX_DATASTORE_VALUE_LENGTH
-          ) {
-            entry.excludeFromIndexes = true;
-          } else {
-            continue;
```

Comment:
We don't want this `continue` because then the `findLargeProperties_` method call below won't happen and we won't search nested properties of `entry`  for large properties and add them to the list of properties to exclude.